database/mssql/openshift/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mssql-rhel-server:2019-latest
+
+RUN mkdir -p /usr/config
+WORKDIR /usr/config
+COPY . /usr/config
+
+USER root
+RUN chmod -R g=u /etc/passwd
```

Comment:
This is required because of the Microsoft bug where it cannot run `sqlcmd` in an OpenShift container.  OpenShift assigns a random UID user account to the pod, which Microsoft doesn't like.